src/pkg/services/m365/m365.go

```diff
@@ -313,6 +313,82 @@ func SitesMap(
 	return idname.NewCache(itn), nil
 }
 
+// ---------------------------------------------------------------------------
+// Teams
```

Comment:
this file is getting a bit long.  Can we move groups/teams to a new file, please?